packages/components/src/components/Address/Address.stories.tsx

```diff
@@ -2,13 +2,22 @@ import { storiesOf } from '@storybook/react';
 import React from 'react';
 import { Address } from '.';
 
-import { useWallet } from '@web3-ui/hooks';
+import { useWallet, Provider } from '@web3-ui/hooks';
+import { Button } from '@chakra-ui/react';
 
-storiesOf('Address', module).add('Default', () => <Address />);
+storiesOf('Address', module).add('Default', () => <Address value='This is an input' />);
```

Comment:
minor nit, but i like my storybook components to contain real looking content. So something like this might look better (and make it more clear what the component is for)
```suggestion
storiesOf('Address', module).add('Default', () => <Address value='testaddress.eth' />);
```